api/nodes/views.py

```diff
@@ -1252,19 +1252,7 @@ def postprocess_query_param(self, key, field_name, operation):
             operation['value'] = Guid.load(operation['value'])
 
     def get_queryset(self):
-        comments = self.get_queryset_from_request()
-        for comment in comments:
-            # Deleted root targets still appear as tuples in the database,
-            # but need to be None in order for the query to be correct.
-            if comment.root_target:
-                if hasattr(comment.root_target.referent, 'is_deleted') and comment.root_target.referent.is_deleted:
-                    comment.root_target = None
-                    comment.save()
-                # Temporary while there are both 'is_deleted' and 'deleted' attributes on referents
-                if comment.root_target and hasattr(comment.root_target.referent, 'deleted') and comment.root_target.referent.deleted:
-                    comment.root_target = None
-                    comment.save()
-        return comments
+        return self.get_queryset_from_request()
```

Comment:
Do we want to show deleted comments in the API? Also I think we can remove all get_queryset if we are just removing the content of the function.

Reply:
Deleted comments are not supposed to be showed in API. I will remove the function.

Reply:
I'm still able to see deleted comments in the API:
<img width="1436" alt="screen shot 2018-04-19 at 2 48 39 pm" src="https://user-images.githubusercontent.com/9688518/39011913-da52cf0e-43e0-11e8-9727-cb15b8a725cb.png">

Reply:
Before this change, the deleted comment is showing up in api. Disregard of my previous response as I don't mean to change this behavior.